doc/reference/lora_lorawan/index.rst

```diff
@@ -1,26 +1,53 @@
-.. _lorawan_api:
+.. _lora_lorawan_api:
 
-LoRaWAN
-#######
+LoRa and LoRaWAN
+################
 
 Overview
 ********
 
-Long Range Wide Area Network (LoRaWAN) is a network layer protocol based on
-the LoRa PHY.
+LoRa (abbrev. for Long Range) is a proprietary low-power wireless
+communication protocol developed by the `Semtech Corporation`_.
+
+LoRa defines the physical layer of the protocol stack, which uses
+chirp-spread-spectrum (CSS) modulation.
+
+LoRaWAN (for Long Range Wide Area Network) defines a networking layer
```

Comment:
Long Range Wide Area Network (LoRaWAN) is the networking layer that works on top of LoRa PHY.

Reply:
Same as above, I'd like to keep it as is.

Reply:
Here, `Long Range Wide Area Network` always corresponds to `LoRaWAN`. But if you want to keep the same pattern, I'm with it.